cypress/integration/finance/create_GL_journal_type_spec.js

```diff
@@ -1,3 +1,25 @@
+let description="test";
+let category="Standard";
```

Comment:
I think the values initialisation can be deleted from here

Reply:
done.